recipe/meta.yaml

```diff
@@ -38,11 +38,10 @@ requirements:
     - jedi >=0.16
     - matplotlib-inline
     - pexpect >4.3    # [unix]
-    - pickleshare
```

Comment:
This was made optional in `ipython` version `8.17.0` with PR: https://github.com/ipython/ipython/pull/14217

Reply:
After discussion below, `pickleshare` was readded: https://github.com/conda-forge/ipython-feedstock/pull/210#discussion_r1500900826